ee/updatemanifest/run.go

```diff
@@ -101,6 +102,7 @@ func run() error {
 			for _, pred := range preds {
 				ns, attr, err := parseNsAttr(pred)
 				if err != nil {
+					parsedPreds = append(parsedPreds, pred)
```

Comment:
Why is this required?

Reply:
As we are overwriting the manifest, I think it should be a good idea to only fix what the tool can fix. It is better to leave the erroneous predicate in the manifest so that the user can fix/remove it, rather than the tool removing it.